src/language-css/printer-postcss.js

```diff
@@ -550,6 +550,22 @@ function genericPrint(path, options, print) {
           continue;
         }
 
+        // Ignore SCSS @forward wildcard suffix
+        if (
+          insideAtRuleNode(path, "forward") &&
+          iNode.type === "value-word" &&
+          iNode.value &&
+          iNode.value.endsWith("-") &&
+          iPrevNode.type === "value-word" &&
+          iPrevNode.value &&
+          iPrevNode.value === "as" &&
+          iNextNode.type === "value-operator" &&
+          iNextNode.value &&
+          iNextNode.value === "*"
```

Comment:
@alexander-akait Could you check if this condition works file? Sorry I'm not familiar with SCSS...

Reply:
I think fine

Reply:
Ideally we need special logic for `@use`/`@forward`, but it will require parser for this, postcss-values-parser violate spec in many places, to solve it in right way we need another parser, so I think we can safely merge it

Reply:
@sosukesuzuki @alexander-akait check is based on [official documentation on prefixes](https://sass-lang.com/documentation/at-rules/forward#adding-a-prefix). `@forward` is available only for Sass files.